src/main/java/com/github/feroult/gapi/spreadsheet/SpreadsheetAPIImpl.java

```diff
@@ -59,36 +44,40 @@ private void resetWorksheet() {
 
 	@Override
 	public void setValue(int i, int j, String value) {
-		try {
-			CellFeed cellFeed = spreadsheetService.getFeed(worksheet.getCellFeedUrl(), CellFeed.class);
+		List<List<Object>> values = Collections.singletonList(
+				Collections.singletonList(value)
+		);
+
+		ValueRange body = new ValueRange().setValues(values);
 
-			CellEntry cellEntry = new CellEntry(i, j, value);
-			cellFeed.insert(cellEntry);
-		} catch (ServiceException | IOException e) {
+		String range = worksheet.getProperties().getTitle() + "!" + (char)(64 + j) + i;
```

Comment:
Question: when equals 1 it means letter A? If so, what do you think about naming them as column and row?

Reply:
I think it's a good idea.